### PR TITLE
feat: Add support for data pipeline load testing

### PR DESCRIPTION
**Pull request by bmtcril**

### CODE REVIEW

File: tutoraspects/templates/aspects/apps/vector/local.toml [hunk 1]
```diff
@@ -8,6 +8,6 @@ type = "docker_logs"
 [transforms.openedx_containers]
 type = "filter"
 inputs = ["docker_logs"]
-condition = 'includes(["lms", "cms", "lms-job", "cms-job"], .label."com.docker.compose.service")'
+condition = 'includes(["lms", "cms", "lms-job", "cms-job", "tracking-consumer"], .label."com.docker.compose.service")'
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/apps/vector/local.toml:11]: This allows Vector to read from the redis bus consumer

File: tutoraspects/patches/openedx-dockerfile-post-python-requirements [hunk 2]
```diff
@@ -1,4 +1,4 @@
 RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared \
-    pip install "platform-plugin-aspects==0.4.0"
+    pip install "git+https://github.com/openedx/platform-plugin-aspects.git@bmtcril/tracking_event_command"
```
Comment (bmtcril) [inline tutoraspects/patches/openedx-dockerfile-post-python-requirements:2]: Needs updated when https://github.com/openedx/platform-plugin-aspects/pull/6 is released

File: tutoraspects/patches/openedx-common-settings [hunk 3]
```diff
@@ -13,6 +13,9 @@ SUPERSET_CONFIG = {
 }
 EVENT_SINK_CLICKHOUSE_PII_MODELS = {{ EVENT_SINK_PII_MODELS }}
 
+EVENT_ROUTING_BACKEND_BATCH_SIZE = 10
+EVENT_ROUTING_BACKEND_BATCHING_ENABLED = False
```
Comment (Ian2012) [inline tutoraspects/patches/openedx-common-settings:17]: I don't think we should modify this variables

File: tutoraspects/plugin.py [hunk 4]
```diff
@@ -136,6 +137,8 @@
         ("ASPECTS_EVENT_SINK_RECENT_BLOCKS_MV", "most_recent_course_blocks_mv"),
         # Vector settings
         ("ASPECTS_DOCKER_HOST_SOCK_PATH", "/var/run/docker.sock"),
+        ("ASPECTS_VECTOR_STORE_TRACKING_LOGS", False),
```
Comment (bmtcril) [inline tutoraspects/plugin.py:140]: These settings are new and let us turn on and off each piece of the Vector storage individually.

File: tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0032_partition_tables_by_year_and_month.py [hunk 5]
```diff
@@ -1,9 +1,5 @@
 """
-Partition the event_sink.user_profile table
-
-.. pii: Stores Open edX user profile data.
-.. pii_types: user_id, name, username, location, phone_number, email_address, birth_date, biography, gender
-.. pii_retirement: local_api, consumer_api
+Partition the xapi table by year and month
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0032_partition_tables_by_year_and_month.py:2]: Everything in this file is just cleaning up incorrect language from the original commit.

**Review by Ian2012 — APPROVED** [review]

**Pull request merged**